### ISSUE TITLE
Compare CNN with MLP in terms of accuracy

### ISSUE THREAD

**Issue by svenvanderburg** [carpentries-incubator/deep-learning-intro#363]
Currently in episode 4 we start out with the statement 'CNNs work better on image data'. But it would be good to actually demonstrate that.
Currently we touch upon MLP for the CIFAR-10 problem in episode 4, but we only do it to show that it would have a lot of parameters compared to a CNN model trained on the same data. 
I suggest to include a small part that trains a simple MLP on the CIFAR-10 dataset, evaluates it, and maybe shows that we are overfitting because there are so many parameters. We can add this as an infobox 'How does this compare to a simple dense neural network?' right after we evaluated the CNN. We can add instructor notes to do it as a demo to save time.
